static/panes/conformance-view.ts

```diff
@@ -88,7 +88,7 @@ export class Conformance extends Pane<ConformanceViewState> {
     constructor(hub: Hub, container: Container, state: PaneState & ConformanceViewState) {
         super(hub, container, state);
         this.compilerService = hub.compilerService;
-        this.maxCompilations = (options.cvCompilerCountMax as number) || 6;
+        this.maxCompilations = (options.cvCompilerCountMax as number) || 6; // TODO
```

Comment:
Will return to this

Reply:
Perhaps put a unique tag name in the TODO for easy comparison vs the many other TODOs. At `$dayjob` I create a GH issue (something like "Finish TODOs for typescript" with a short description and then I use `// TODO(#234)` (where 234 is the GH issue number). then it's bidirectionally greppable and also has a place to argue^H^H^H^H^Hdiscuss further.